test_transformer_node:
  ros__parameters:
    static_transform_names: ["map_to_odom", "odom_to_base"]

    static_transforms:
      map_to_odom:
        parent_frame: "map"
        child_frame: "odom"
        translation: { x: 0.1, y: 0.2, z: 0.0 }
        rotation: { x: 0.0, y: 0.0, z: 0.0, w: 1.0 }
      odom_to_base:
        parent_frame: "odom"
        child_frame: "base_link"
        translation: { x: 0.5, y: 0.0, z: 0.1 }
        rotation: { x: 0.0, y: 0.0, z: 0.0, w: 1.0 }

    boundary_frame_names: ["map", "base_link"]

    boundaries:
      map:
        min_x: -50.0
        min_y: -50.0
        min_z: -5.0
        max_x: 50.0
        max_y: 50.0
        max_z: 5.0
      base_link:
        min_x: -2.0
        min_y: -2.0
        min_z: -0.5
        max_x: 2.0
        max_y: 2.0
        max_z: 2.0

    target_frame: "map"
